### PR TITLE
Rifki/feature/enhance validasi data

### PR DESCRIPTION
**Pull request by rifkiridha**

### CODE REVIEW

File: src/components/organisms/ValidasiData/index.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+        receivedData += new TextDecoder().decode(value);
+
+        const regex = /(?<=\})\s*(?=\{)/g;
+        const modifiedData = "[" + receivedData.replace(regex, ",") + "]";
+
+        try {
+          const parsedData = await JSON.parse(modifiedData);
+          setData(parsedData);
+          // console.log("parsed data for ch id : ", channelId, " : ", parsedData);
```
Comment (faisalrahmatdani) [inline src/components/organisms/ValidasiData/index.jsx:37]: apus aja commentnya

File: src/components/molecules/CardValidasiData/index.jsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  const [nik, setNik] = useState(null);
+  const [ktpData, setKtpData] = useState(null);
+
+  const handleClose = () => {
+    setModalShow(!modalShow);
+  };
+
+  useEffect(() => {
+    if (externalId) {
+      // console.log("fetching");
```
Comment (faisalrahmatdani) [inline src/components/molecules/CardValidasiData/index.jsx:18]: delete

File: src/components/molecules/CardValidasiData/index.jsx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+            headers: {
+              Authorization: `Bearer ${token}`,
+            },
+          });
+
+          if (response.status === 200) {
+            const result = response.data;
+            if (result.data) {
+              setKtpData(result.data);
+              // console.log("result", result.data);
```
Comment (faisalrahmatdani) [inline src/components/molecules/CardValidasiData/index.jsx:34]: delete

File: src/components/organisms/ValidasiData/index.jsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+import { baseUrlFromEocortex } from "@/service/eocortex.service";
+
+const FaceRecognitionForValidasi = ({ channelId }) => {
+  const [data, setData] = useState([]);
+
+  let validasiMapping = data?.slice().reverse();
+
+  const fetchValidasiData = async () => {
+    try {
+      // console.log("base url from eocortex :", baseUrlFromEocortex);
```
Comment (faisalrahmatdani) [inline src/components/organisms/ValidasiData/index.jsx:13]: delete comment

File: src/components/molecules/CardValidasiData/index.jsx (hunk 2 above)
Comment (rifkiridha) [reply src/components/molecules/CardValidasiData/index.jsx]: sudah

File: src/components/organisms/ValidasiData/index.jsx (hunk 4 above)
Comment (rifkiridha) [reply src/components/organisms/ValidasiData/index.jsx]: done

**Review by faisalrahmatdani — APPROVED** [review]
LGTM!

**Pull request merged**